eng/scripts/Export-API.ps1

```diff
@@ -3,9 +3,25 @@ param (
     [Parameter(Position=0)]
     [ValidateNotNullOrEmpty()]
     [string] $ServiceDirectory,
-    [string] $SDKType = "all"
+    [string] $SDKType = "all", 
+    [switch] $SpellCheckPublicApiSurface
 )
 
+if ($SpellCheckPublicApiSurface -and -not (Get-Command 'npx')) { 
+    Write-Error "Could not locate npx. Install NodeJS (includes npm and npx) https://nodejs.org/en/download/"
```

Comment:
Is there a specific minimum node version that needs to be installed for this to work?

Reply:
Yes. You'll need npm >= v5.2 (July 2017 release) -- https://nodejs.dev/learn/the-npx-nodejs-package-runner

Reply:
Might be good to add that to the exception.

Reply:
Captured that in https://github.com/Azure/azure-sdk-tools/issues/1600